serveradmin/nessus/views.py

```diff
@@ -0,0 +1,63 @@
+"""Serveradmin - Nessus Integration
+
+Copyright (c) 2023 InnoGames GmbH
+"""
+from urllib.parse import urlencode
+
+from django.conf import settings
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.http import (
+    HttpResponseBadRequest, HttpResponseServerError
+)
+from django.template.response import TemplateResponse
+from django.views.decorators.csrf import ensure_csrf_cookie
+
+from adminapi.filters import Any
+from serveradmin.dataset import Query
+from serveradmin.nessus.models import NessusAPI
+
+@login_required
+@ensure_csrf_cookie
+def nessus_config(request):
+    """Generate nessus scan configuration page
+
+    :param request:
+    :return:
+    """
+
+    hostnames = [h for h in request.GET.get('hosts', '').split(', ')]
```

Comment:
The scan command in Servershell only uses object_id. Is this a left over from testing or are there other clients calling this URL which pass a hostname ?